.eslintrc

```diff
@@ -1,3 +1,9 @@
 {
-  extends: "lob"
+  extends: "lob",
+  globals: {
+    expect: false
+  },
+  rules: {
+    arrow-body-style: 0
```

Comment:
I had to disable this rule because of an [eslint bug](https://github.com/eslint/eslint/issues/4411) that causes it complain about empty arrow bodies, like [here](https://github.com/lob/hapi-bookshelf-total-count/pull/1/files#diff-0fd0e07cf6d02bf7cf00f18cebb8e6eaR33). It's been fixed in the later eslint version but that version has breaking changes that affect a bunch of other rules. So this is temporary until we update our linter to the latest version of eslint and update our rules accordingly